redaxo/src/core/lib/config_db.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @psalm-readonly
+ */
+class rex_config_db
```

Comment:
```suggestion
final class rex_config_db
```

Allgemeine Frage: Wollen wir zukünftig vielleicht bei Neuklassen vermehrt `final` nutzen?
Hat den Vorteil, dass man die leichter später ändern kann, da man bzgl. BC eben die Vererbung nicht berücksichtigen muss.

Reply:
Gerne